R/projectInit.R

```diff
@@ -14,16 +14,18 @@ NULL
 #' script for the project. You pass a complete folder or a relative path.
 #'
 #' @param codeRoot Path to the folder representing a code repository root.
-#' @param dataDir Path to folder containing project data.
+#' @param dataDir Path to folder containing processed project data.
+#' @param rawDir Path to folder containing raw project data.
 #' @param outputSubdir Location for project-specific output, resolved by 
 #'                     \code{dirOut} and stored as option \code{ROUT.SUBDIR}.
 #' @param resources Location of general-purpose resources; default is to use 
 #'                  system environment variable \code{RESOURCES}.
 #' @param scriptSubdir Name for the folder within \code{codeRoot} that 
 #'                     stores the scripts for this project.
+#' @param subproject name of the subproject to be activated
 #' @export
-projectInit = function(codeRoot=NULL, dataDir=NULL, outputSubdir=NULL,
-						resources=Sys.getenv("RESOURCES"), scriptSubdir="src") {
+projectInit = function(codeRoot=NULL, dataDir=NULL, rawDir=NULL, outputSubdir=NULL,
```

Comment:
On `.updateSubconfig` the default subproject is `NULL`, and that function does a check on `NULL` but not on an empty character vector. I don't know which may work out better or be required sematically, but it feels safer to make consistent the way that the notion _lack of subproject_ is represented.

Reply:
yes, there's inconsistency in that matter in PEP-related software in `R`. And tbh I don't what was the reason. I'll try to update `pepr` and `BiocProject` accordingly since they use `character()` as well.

Reply:
Nice, looks great!